src/richie/apps/courses/models/category.py

```diff
@@ -26,6 +26,8 @@ class Category(BasePageExtension):
     page that presents the thematic.
     """
 
+    color = models.CharField(max_length=10, blank=True, null=True)
```

Comment:
I think 10 is too short a length. If we want to use eg. colors with transparency like `rgba(255,255,255,0.75)` it would not hurt to have a `max_length` of eg. 25

Reply:
The color is stored as hexadecimal. With transparency it is of the form #24c4a2aa, so 9 characters.

This being said, the default html5 widget I used does not seem to handle transparency and we don't have the use case for it at the moment.